Legacy/real_time/arduino_src/radio_buggy_mega/Makefile

```diff
@@ -109,7 +116,8 @@ EXTRAINCDIRS =
 #     gnu89 = c89 plus GCC extensions
 #     c99   = ISO C99 standard (not yet fully implemented)
 #     gnu99 = c99 plus GCC extensions
-CSTANDARD = -std=gnu99
+CSTANDARD = -std=iso9899:1999
+CPPSTANDARD = -std=c++11

```

Comment:
Which standards do we want to use? I switched the C one without real reason. Should we discuss this?